linux/src/reals_linux/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(azure_c_util_reals_c_files
+    real_constbuffer.c
+    real_constbuffer_array.c
+    real_constbuffer_array_batcher_nv.c
+    real_crt_abstractions.c
+    real_doublylinkedlist.c
+    real_memory_data.c
+    real_singlylinkedlist.c
+    real_threadapi.c
+    real_uuid.c
+)
+
+set(azure_c_util_reals_h_files
+    real_constbuffer.h
+    real_constbuffer_renames.h
+    real_constbuffer_array.h
+    real_constbuffer_array_renames.h
+    real_constbuffer_array_batcher_nv.h
+    real_constbuffer_array_batcher_nv_renames.h
+    real_crt_abstractions.h
+    real_crt_abstractions_renames.h
+    real_doublylinkedlist.h
+    real_doublylinkedlist_renames.h
+    real_memory_data.h
+    real_memory_data_renames.h
+    real_singlylinkedlist.h
```

Comment:
All of these can be removed (only need uuid/threadapi for now). Same above and same for the files themselves

Reply:
I think uuid is also in azure-c-util so im gonna remove it.

---
In reply to: [439587768](https://github.com/Azure/azure-c-pal/pull/2#discussion_r439587768) [](ancestors = 439587768)

Reply:
Yes, you are right

---
In reply to: [439589444](https://github.com/Azure/azure-c-pal/pull/2#discussion_r439589444) [](ancestors = 439589444,439587768)